packs/docker/actions/lib/docker_wrapper.py

```diff
@@ -52,6 +52,24 @@ def build(self, path=None, fileobj=None, tag=None):
                 json_output = six.advance_iterator(result)
         except StopIteration:
             pass
-        except Exception, e:
+        except Exception as e:
+            sys.stderr.write('Error: %s' % (str(e)))
+            raise e
+
+    def push(self, repo, tag=None, insecure_registry=False):
+        try:
+            for line in self._client.push(repo, tag=tag, insecure_registry=insecure_registry,
```

Comment:
Yeah, streaming is good, now we just need to implement action execution stdout and stderr streaming API ourselves :P